ui/pom.xml

```diff
@@ -13,7 +13,7 @@
 
     <properties>
         <node.version>v12.18.3</node.version>
-        <yarn.version>v1.12.1</yarn.version>
+        <yarn.version>v1.22.5</yarn.version>
```

Comment:
I have just put up a pull request to replace the deleted yarn.lock 
My version of yarn is currently 1.22.10

Reply:
OK updated